module/batch-action/src/Domain/Event/BatchActionStoppedEvent.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/**
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Ergonode\BatchAction\Domain\Event;
+
+use Ergonode\BatchAction\Domain\Entity\BatchActionId;
+use Ergonode\SharedKernel\Domain\DomainEventInterface;
+
+class BatchActionStoppedEvent implements DomainEventInterface
```

Comment:
```suggestion
class BatchActionWaitingForDecisionEvent implements DomainEventInterface
```
perhaps? Less things to "understand"